.gitignore

```diff
@@ -0,0 +1,5 @@
+﻿################################################################################
+# This .gitignore file was automatically created by Microsoft(R) Visual Studio.
+################################################################################
+
+/.vs/StudyTaskApi/v16/.suo
```

Comment:
Please edit this file by hand to exclude *.sqlite files. Those are your VS settings and shouldn't be checked into source control.